README.md

```diff
@@ -494,14 +500,19 @@ PaymentInitiationResponse response = paymentClient.sendOnlinePayment(paymentInit
 
 The payment details request requires the following parameters:
 - `storeId` - the ID of your assigned store
-- `pageCursor` - the Order ID returned in the initial payment response, a unique UUID value for each payment.
+- `pageCursor` - set pagination settings
```

Comment:
Please explain how to use this, for example: `page = 1` and `pageSize = 1` means only return the first payment
(Assuming that is correct)